x-pack/plugins/cases/common/utils/user_actions.ts

```diff
@@ -71,3 +72,7 @@ export const isCreateCaseUserAction = (userAction: unknown): userAction is Creat
 
 export const isUserActionType = (field: string): field is UserActionTypes =>
   ActionTypes[field as UserActionTypes] != null;
+
+export const isCategoryUserAction = (userAction: unknown): userAction is CategoryUserAction =>
+  (userAction as CategoryUserAction)?.type === ActionTypes.category &&
+  (userAction as CategoryUserAction)?.payload?.category !== undefined;
```

Comment:
Is this enough? should we check if it is `string | null` also @cnasikas

Reply:
Yes, it is enough. It should not be possible to have any other value there.